### DISCUSSION TITLE
Nobara 39

### DISCUSSION THREAD

**Discussion by Rick-1242** [JaKooLit/Fedora-Hyprland#40] [Q&A] [answered]
What makes this brake with Nobara 39?

**Comment by JaKooLit** [comment]
what?

**Comment by Gerotama** [comment]
> What makes this brake with Nobara 39?

I'll quote what JaKooLit says on yt: Some said not work on nobara 39 because nobara moved to dnf5

**Comment by JaKooLit** [comment]
Somebody posted on facebook that he managed to install Hyprland using this script on Nobara 39 

He said he didnt change anything on the script

**Comment by JaKooLit** [comment, marked as answer]
https://github.com/JaKooLit/screenshots/blob/main/Distro-Hyprland/Fedora/nobara39.png

It works with Nobara 39.. You just need to edit the nwg-look.sh script

Refer to readme
